src/Microsoft.Data.Sqlite/SqliteConnectionStringBuilder.cs

```diff
@@ -14,27 +14,32 @@ namespace Microsoft.Data.Sqlite
     public class SqliteConnectionStringBuilder : DbConnectionStringBuilder
     {
         private const string DataSourceKeyword = "Data Source";
+        private const string CacheKeyword = "Cache";
 
         private enum Keywords
         {
-            DataSource
+            DataSource,
+            Cache
         }
 
         private static readonly IReadOnlyList<string> _validKeywords;
         private static readonly IReadOnlyDictionary<string, Keywords> _keywords;
 
         private string _dataSource = string.Empty;
+        private CacheMode _cacheMode = CacheMode.Private;
```

Comment:
There are subtle differences between unspecified and Private:

> Shared cache is disabled by default. But this might change in future releases of SQLite.

Maybe we should add a `Default` option.